bundles/org.openhab.binding.easee/src/main/java/org/openhab/binding/easee/internal/model/ChargerOpState.java

```diff
@@ -0,0 +1,49 @@
+package org.openhab.binding.easee.internal.model;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+@NonNullByDefault
+public enum ChargerOpState {
+    OFFLINE(0),
+    DISCONNECTED(1),
+    WAITING(2),
+    CHARGING(3),
+    COMPLETED(4),
+    ERROR(5),
+    READY_TO_CHARGE(6),
+    NOT_AUTHENTICATED(7),
+    DEAUTHENTICATING(8),
+    UNKNOWN_STATE(-1);
+
+    private final int code;
+
+    private ChargerOpState(int code) {
+        this.code = code;
+    }
+
+    public boolean isAuthenticatedState() {
+        switch (this) {
+            case WAITING:
+            case CHARGING:
+            case COMPLETED:
+            case ERROR:
```

Comment:
Are you sure 'ERROR' is an authenticated and connected state?

Reply:
not sure but most likely I expect any errors during process of charging. I never had this state and there is no documentation. It just says "Error in Charger" as description. Well as this is a state that usually does not occur I could also remove it from the authenticated states list.

Reply:
according to the "reason for no current" enum which has more detailed states it looks like that error could occur in many situations. Most are not caused by the car. I will change the code accordingly.